README.md

```diff
@@ -0,0 +1,42 @@
+# MiniTwit
+
+MiniTwit is an messaging application being converted from Flask to FastAPI
+
+
+# How to dockerise the app.
+To run the application, we suggest using docker and building an image of the app.
+For this purpose we created a docker-compose.yml file. 
+
+Instructions on how to build the docker image:
+- Locate the docker-compose.yml file inside src directory 
+- On VS code: Right click and select "Compose Up" and the image will be built and ran inside docker. 
+
+Alternatevely, you can use terminal commands to achieve the same thing.
+
+>  docker-compose build
+
+>  docker-compose up 
+
+
+# How to install dependencies
```

Comment:
Is this part necessary since the app is dockerised?

Reply:
Push it with a mention of it only being needed outside of dockerization